ci/spack.yaml

```diff
@@ -6,20 +6,20 @@ spack:
       - intel
       - gcc@10:10
   specs:
-  - netcdf-c@4.7.4
-  - netcdf-fortran@4.5.3
+  - netcdf-c@4.9.2
+  - netcdf-fortran@4.6.1
   - bacio@2.4.1
-  - w3emc@2.9.2
+  - w3emc@2.10.0
   - g2@3.4.5
-  - g2tmpl@1.10.0
-  - sp@2.3.3
-  - ip@3.3.3
+  - g2tmpl@1.10.2
+  - sp@2.5.0
+  - ip@4.3.0
   - sigio@2.3.2
   - sfcio@1.4.1
   - nemsio@2.5.4
   - wrf-io@1.2.0
-  - crtm@2.3.0
+  - crtm@2.4.0.1
   view: true
   concretizer:
-    unify: true
+    unify: when_possible
 
```

Comment:
Is there a reason to switching to 'when_possible'? If there is more than one version of a package getting installed, then there will be trouble with `view: true` set, so unless I'm missing something I would suggest reverting to `unify: true`.